packages/cloud_firestore/cloud_firestore_platform_interface/lib/src/method_channel/method_channel_firestore.dart

```diff
@@ -305,30 +161,31 @@ class MethodChannelFirebaseFirestore extends FirebaseFirestorePlatform {
 
   @override
   Stream<void> snapshotsInSync() {
-    int handle = MethodChannelFirebaseFirestore.nextMethodChannelHandleId;
-
-    StreamController<QuerySnapshotPlatform /*!*/ >
-        controller; // ignore: close_sinks
-    controller = StreamController<QuerySnapshotPlatform /*!*/ >.broadcast(
-      onListen: () {
-        MethodChannelFirebaseFirestore.snapshotInSyncObservers[handle] =
-            controller /*!*/;
-        MethodChannelFirebaseFirestore.channel.invokeMethod<int>(
-          'Firestore#addSnapshotsInSyncListener',
+    StreamSubscription<dynamic> snapshotStream;
+    StreamController<void> controller; // ignore: close_sinks
```

Comment:
Could we copy the comment from the other function explaining why we can ignore the lint here too?